package.json

```diff
@@ -62,5 +62,44 @@
     "temp": "0.8.3",
     "uuid": "3.0.1",
     "xml-escape": "1.1.0"
+  },
+  "keywords": [
+    "nativescript",
+    "nativescript:extension",
+    "nativescript-cloud",
+    "cloud",
+    "build"
+  ],
+  "nativescript": {
+    "commands": [
+      "accept|eula",
+      "account",
+      "account|features",
+      "account|usage",
+      "build|cloud",
+      "cloud|build",
+      "cloud|clean|workspace",
+      "cloud|codesign",
+      "cloud|deploy",
+      "cloud|lib|version",
+      "cloud|publish|android",
+      "cloud|publish|ios",
```

Comment:
maybe missing "publish|cloud" or it is not supported?

Reply:
It is not supported